processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2022-02-23 09:06:09.634927789 -0500
+++ memcpu.hex	2022-02-23 09:06:09.558924944 -0500
@@ -81,133 +81,133 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
+:0400C1000000000437
 :0400C2000000000337
-:0400C3000000000435
+:0400C3000000000138
 :0400C4000000000731
 :0400C5000000000B2C
 :0400C6000000000C2A
 :0400C7000000001520
 :0400C800000000161E
 :0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
+:0400CA000000002909
+:0400CB000000001D14
+:0400CC00000000210F
+:0400CD000000001B14
+:0400CE00000000200E
+:0400CF000000001914
+:0400D00000000050DC
+:0400D1000000003EED
+:0400D2000000004BDF
+:0400D30000000047E2
+:0400D4000000004DDB
+:0400D50000000042E5
+:0400D6000000003DE9
+:0400D70000000036EF
+:0400D8000000003DE7
+:0400D90000000038EB
+:0400DA0000000033EF
+:0400DB0000000053CE
 :0400DC0000000051CF
-:0400DD0000000053CC
+:0400DD0000000051CE
 :0400DE000000005AC4
 :0400DF0000000063BA
 :0400E00000000064B8
 :0400E1000000000714
 :0400E2000000000713
 :0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
+:0400E4000000000F09
+:0400E50000000020F7
+:0400E600000000090D
+:0400E7000000002FE6
+:0400E8000000001103
+:0400E90000000023F0
+:0400EA0000000020F2
+:0400EB000000000A07
 :0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
+:0400ED0000000028E7
+:0400EE000000001AF4
+:0400EF000000000805
+:0400F00000000024E8
+:0400F10000000026E5
+:0400F2000000000802
 :0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
+:0400F40000000010F8
+:0400F50000000008FF
+:0400F60000000041C5
 :0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
+:0400F80000000030D4
+:0400F9000000005CA7
+:0400FA0000000059A9
 :0400FB0000000044BD
-:0400FC0000000059A7
+:0400FC000000005CA4
 :0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
+:0400FE0000000043BB
+:0400FF0000000042BB
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
+:0401400000000007B4
+:0401410000000004B6
+:0401420000000003B6
+:0401430000000001B7
 :0401440000000007B0
 :0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
+:040146000000000FA6
+:0401470000000007AD
+:040148000000000BA8
+:040149000000000CA6
+:04014A00000000159C
+:04014B00000000169A
+:04014C000000001699
+:04014D00000000208E
+:04014E0000000009A4
+:04014F000000002F7D
+:040150000000002982
+:040151000000001199
+:040152000000002386
+:040153000000001D8B
+:040154000000002087
+:040155000000000A9C
+:04015600000000178E
+:04015700000000287C
+:040158000000002182
+:040159000000001A88
+:04015A000000000899
+:04015B00000000247C
+:04015C000000001B84
+:04015D00000000207E
+:04015E000000001984
 :04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
+:040160000000000893
+:040161000000002674
+:040162000000001089
+:040163000000000890
+:040164000000004156
 :040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
+:040166000000003065
+:040167000000005C38
+:040168000000005043
+:040169000000003E54
+:04016A000000004B46
+:04016B000000004749
+:04016C000000004D42
 :04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
+:04016E000000003D50
+:04016F000000003656
+:040170000000003D4E
+:040171000000003852
+:040172000000003356
+:040173000000005335
+:040174000000005136
 :040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
+:04017600000000592C
+:040177000000004440
+:040178000000005C27
 :040179000000005A28
 :04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:04017B00000000433D
+:04017C00000000423D
+:04017D00000000641A
+:04017E00000000631A
+:04017F000000006319
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/ld_hazard.asm (1 of 1)
processing file asmFiles/beq.asm (1 of 24)
processing file asmFiles/beq_nt.asm (2 of 24)
processing file asmFiles/bne_nt.asm (3 of 24)
processing file asmFiles/bne_taken.asm (4 of 24)
processing file asmFiles/branch.asm (5 of 24)
processing file asmFiles/control_flow_hazard.asm (6 of 24)
processing file asmFiles/count_days.asm (7 of 24)
processing file asmFiles/empty.asm (8 of 24)
processing file asmFiles/fib.asm (9 of 24)
--- memsim.hex	2022-02-23 09:06:41.719128975 -0500
+++ memcpu.hex	2022-02-23 09:06:41.635125830 -0500
@@ -14,25 +14,25 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400240000000002D6
+:0400250000000004D3
+:0400260000000004D2
+:0400270000000008CD
+:0400280000000008CC
+:0400290000000010C3
+:04002A0000000010C2
+:04002B0000000020B1
+:04002C0000000020B0
+:04002D00000000408F
+:04002E00000000408E
+:04002F00000000804D
+:04003000000000804C
+:0400310000000100CA
+:0400320000000100C9
+:0400330000000200C7
+:0400340000000200C6
+:0400350000000400C3
+:0400360000000400C2
+:0400370000000800BD
 :0403C0000000001623
 :00000001FF
processing file asmFiles/forward.asm (10 of 24)
processing file asmFiles/halt.asm (11 of 24)
processing file asmFiles/jreg.asm (12 of 24)
processing file asmFiles/jump.asm (13 of 24)
processing file asmFiles/ld_hazard.asm (14 of 24)
processing file asmFiles/mergesort.asm (15 of 24)
--- memsim.hex	2022-02-23 09:06:51.640500417 -0500
+++ memcpu.hex	2022-02-23 09:06:51.562497498 -0500
@@ -81,133 +81,133 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
+:0400C1000000000437
 :0400C2000000000337
-:0400C3000000000435
+:0400C3000000000138
 :0400C4000000000731
 :0400C5000000000B2C
 :0400C6000000000C2A
 :0400C7000000001520
 :0400C800000000161E
 :0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
+:0400CA000000002909
+:0400CB000000001D14
+:0400CC00000000210F
+:0400CD000000001B14
+:0400CE00000000200E
+:0400CF000000001914
+:0400D00000000050DC
+:0400D1000000003EED
+:0400D2000000004BDF
+:0400D30000000047E2
+:0400D4000000004DDB
+:0400D50000000042E5
+:0400D6000000003DE9
+:0400D70000000036EF
+:0400D8000000003DE7
+:0400D90000000038EB
+:0400DA0000000033EF
+:0400DB0000000053CE
 :0400DC0000000051CF
-:0400DD0000000053CC
+:0400DD0000000051CE
 :0400DE000000005AC4
 :0400DF0000000063BA
 :0400E00000000064B8
 :0400E1000000000714
 :0400E2000000000713
 :0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
+:0400E4000000000F09
+:0400E50000000020F7
+:0400E600000000090D
+:0400E7000000002FE6
+:0400E8000000001103
+:0400E90000000023F0
+:0400EA0000000020F2
+:0400EB000000000A07
 :0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
+:0400ED0000000028E7
+:0400EE000000001AF4
+:0400EF000000000805
+:0400F00000000024E8
+:0400F10000000026E5
+:0400F2000000000802
 :0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
+:0400F40000000010F8
+:0400F50000000008FF
+:0400F60000000041C5
 :0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
+:0400F80000000030D4
+:0400F9000000005CA7
+:0400FA0000000059A9
 :0400FB0000000044BD
-:0400FC0000000059A7
+:0400FC000000005CA4
 :0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
+:0400FE0000000043BB
+:0400FF0000000042BB
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
+:0401400000000007B4
+:0401410000000004B6
+:0401420000000003B6
+:0401430000000001B7
 :0401440000000007B0
 :0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
+:040146000000000FA6
+:0401470000000007AD
+:040148000000000BA8
+:040149000000000CA6
+:04014A00000000159C
+:04014B00000000169A
+:04014C000000001699
+:04014D00000000208E
+:04014E0000000009A4
+:04014F000000002F7D
+:040150000000002982
+:040151000000001199
+:040152000000002386
+:040153000000001D8B
+:040154000000002087
+:040155000000000A9C
+:04015600000000178E
+:04015700000000287C
+:040158000000002182
+:040159000000001A88
+:04015A000000000899
+:04015B00000000247C
+:04015C000000001B84
+:04015D00000000207E
+:04015E000000001984
 :04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
+:040160000000000893
+:040161000000002674
+:040162000000001089
+:040163000000000890
+:040164000000004156
 :040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
+:040166000000003065
+:040167000000005C38
+:040168000000005043
+:040169000000003E54
+:04016A000000004B46
+:04016B000000004749
+:04016C000000004D42
 :04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
+:04016E000000003D50
+:04016F000000003656
+:040170000000003D4E
+:040171000000003852
+:040172000000003356
+:040173000000005335
+:040174000000005136
 :040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
+:04017600000000592C
+:040177000000004440
+:040178000000005C27
 :040179000000005A28
 :04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:04017B00000000433D
+:04017C00000000423D
+:04017D00000000641A
+:04017E00000000631A
+:04017F000000006319
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (16 of 24)
processing file asmFiles/mult_procedure.asm (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/test.branchjump1.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
processing file asmFiles/try.asm (24 of 24)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2022-02-23 09:07:21.747627584 -0500
+++ memcpu.hex	2022-02-23 09:07:21.672624777 -0500
@@ -14,25 +14,25 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400240000000002D6
+:0400250000000004D3
+:0400260000000004D2
+:0400270000000008CD
+:0400280000000008CC
+:0400290000000010C3
+:04002A0000000010C2
+:04002B0000000020B1
+:04002C0000000020B0
+:04002D00000000408F
+:04002E00000000408E
+:04002F00000000804D
+:04003000000000804C
+:0400310000000100CA
+:0400320000000100C9
+:0400330000000200C7
+:0400340000000200C6
+:0400350000000400C3
+:0400360000000400C2
+:0400370000000800BD
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2022-02-23 09:07:26.908820812 -0500
+++ memcpu.hex	2022-02-23 09:07:26.833818004 -0500
@@ -14,25 +14,25 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400240000000002D6
+:0400250000000004D3
+:0400260000000004D2
+:0400270000000008CD
+:0400280000000008CC
+:0400290000000010C3
+:04002A0000000010C2
+:04002B0000000020B1
+:04002C0000000020B0
+:04002D00000000408F
+:04002E00000000408E
+:04002F00000000804D
+:04003000000000804C
+:0400310000000100CA
+:0400320000000100C9
+:0400330000000200C7
+:0400340000000200C6
+:0400350000000400C3
+:0400360000000400C2
+:0400370000000800BD
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2022-02-23 09:23:14.725293186 -0500
+++ memcpu.hex	2022-02-23 09:23:14.626289482 -0500
@@ -13,26 +13,26 @@
 :04000C00FFFFFFFFF4
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400240000000002D6
+:0400250000000003D4
+:0400260000000004D2
+:0400270000000006CF
+:0400280000000009CB
+:040029000000000DC6
+:04002A0000000013BF
+:04002B000000001CB5
+:04002C0000000029A7
+:04002D000000003C93
+:04002E000000005876
+:04002F00000000814C
+:04003000000000BD0F
+:0400310000000115B5
+:040032000000019633
+:040033000000025374
+:04003400000003685D
+:04003500000004FEC5
+:04003600000007516E
+:0400370000000AB902
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2022-02-23 09:49:59.841389302 -0500
+++ memcpu.hex	2022-02-23 09:49:59.751385933 -0500
@@ -13,26 +13,26 @@
 :04000C00FFFFFFFFF4
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400240000000001D7
+:0400250000000001D6
+:0400260000000001D5
+:0400270000000001D4
+:0400280000000001D3
+:0400290000000001D2
+:04002A0000000001D1
+:04002B0000000001D0
+:04002C0000000001CF
+:04002D0000000001CE
+:04002E0000000001CD
+:04002F0000000001CC
+:0400300000000001CB
+:0400310000000001CA
+:0400320000000001C9
+:0400330000000001C8
+:0400340000000001C7
+:0400350000000001C6
+:0400360000000001C5
+:0400370000000001C4
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2022-02-23 09:51:06.401881101 -0500
+++ memcpu.hex	2022-02-23 09:51:06.293877057 -0500
@@ -13,26 +13,26 @@
 :04000C00FFFFFFFFF4
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400240000000001D7
+:0400250000000001D6
+:0400260000000001D5
+:0400270000000001D4
+:0400280000000001D3
+:0400290000000001D2
+:04002A0000000001D1
+:04002B0000000001D0
+:04002C0000000001CF
+:04002D0000000001CE
+:04002E0000000001CD
+:04002F0000000001CC
+:0400300000000001CB
+:0400310000000001CA
+:0400320000000001C9
+:0400330000000001C8
+:0400340000000001C7
+:0400350000000001C6
+:0400360000000001C5
+:0400370000000001C4
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2022-02-23 09:52:27.248907738 -0500
+++ memcpu.hex	2022-02-23 09:52:27.198905866 -0500
@@ -13,26 +13,26 @@
 :04000C00FFFFFFFFF4
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400240000000002D6
+:0400250000000003D4
+:0400260000000004D2
+:0400270000000006CF
+:0400280000000009CB
+:040029000000000DC6
+:04002A0000000013BF
+:04002B000000001CB5
+:04002C0000000029A7
+:04002D000000003C93
+:04002E000000005876
+:04002F00000000814C
+:04003000000000BD0F
+:0400310000000115B5
+:040032000000019633
+:040033000000025374
+:04003400000003685D
+:04003500000004FEC5
+:04003600000007516E
+:0400370000000AB902
 :0403C0000000001623
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2022-02-23 10:34:21.088026675 -0500
+++ memcpu.hex	2022-02-23 10:34:20.992023081 -0500
@@ -14,25 +14,25 @@
 :0400210000000001DA
 :0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400240000000002D6
+:0400250000000004D3
+:0400260000000004D2
+:0400270000000008CD
+:0400280000000008CC
+:0400290000000010C3
+:04002A0000000010C2
+:04002B0000000020B1
+:04002C0000000020B0
+:04002D00000000408F
+:04002E00000000408E
+:04002F00000000804D
+:04003000000000804C
+:0400310000000100CA
+:0400320000000100C9
+:0400330000000200C7
+:0400340000000200C6
+:0400350000000400C3
+:0400360000000400C2
+:0400370000000800BD
 :0403C0000000001623
 :00000001FF
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2022-02-23 10:39:13.789983626 -0500
+++ memcpu.hex	2022-02-23 10:39:13.678979470 -0500
@@ -81,131 +81,131 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
+:0400C1000000000338
+:0400C2000000003802
+:0400C3000000003DFC
+:0400C4000000000434
+:0400C5000000003601
+:0400C6000000000135
+:0400C7000000002015
+:0400C8000000003DF7
+:0400C90000000042F1
+:0400CA000000001B17
+:0400CB000000002110
+:0400CC000000000729
+:0400CD000000001D12
+:0400CE000000000B23
+:0400CF000000001617
+:0400D0000000004DDF
+:0400D10000000051DA
+:0400D20000000053D7
+:0400D30000000047E2
+:0400D4000000004BDD
 :0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
+:0400D6000000000C1A
+:0400D7000000001510
+:0400D800000000160E
+:0400D90000000029FA
 :0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400DB000000001908
+:0400DC0000000033ED
+:0400DD0000000051CE
 :0400DE000000005AC4
 :0400DF0000000063BA
 :0400E00000000064B8
-:0400E1000000000714
+:0400E10000000043D8
 :0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
+:0400E30000000032E7
+:0400E4000000001008
+:0400E50000000026F1
 :0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
+:0400E70000000026EF
+:0400E80000000024F0
+:0400E900000000080B
+:0400EA000000001AF8
+:0400EB0000000028E9
+:0400EC000000005BB5
+:0400ED000000005CB3
+:0400EE0000000017F7
+:0400EF000000000706
+:0400F0000000000A02
+:0400F10000000020EB
+:0400F20000000044C6
+:0400F30000000023E6
+:0400F4000000000701
+:0400F50000000011F6
+:0400F60000000059AD
+:0400F7000000002FD6
+:0400F80000000009FB
+:0400F90000000020E3
+:0400FA0000000041C1
+:0400FB000000005CA5
+:0400FC000000000FF1
+:0400FD0000000008F7
+:0400FE0000000030CE
+:0400FF0000000042BB
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
+:0401400000000003B8
+:040141000000003882
+:040142000000003D7C
+:0401430000000004B4
+:040144000000003681
+:0401450000000001B5
+:040146000000002095
+:040147000000003D77
+:040148000000004271
+:040149000000001B97
+:04014A000000002190
+:04014B0000000007A9
+:04014C000000001D92
+:04014D000000000BA3
+:04014E000000001697
+:04014F000000004369
+:0401500000000007A4
+:040151000000003278
+:040152000000001099
+:040153000000002682
+:04015400000000089F
+:040155000000002680
+:040156000000002481
+:04015700000000089C
+:040158000000001A89
+:04015900000000287A
+:04015A000000004D54
+:04015B00000000514F
+:04015C00000000534C
+:04015D000000004757
+:04015E000000004B52
+:04015F000000003E5E
+:040160000000000C8F
+:040161000000001585
+:040162000000001683
+:04016300000000296F
+:040164000000005047
+:04016500000000197D
 :040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
+:040167000000005143
+:040168000000005A39
+:040169000000005B37
+:04016A000000005C35
+:04016B000000001779
+:04016C000000000788
+:04016D000000000A84
+:04016E00000000206D
+:04016F000000004448
+:040170000000002368
+:040171000000000783
+:040172000000001178
+:04017300000000592F
+:040174000000002F58
+:04017500000000097D
+:040176000000002065
+:040177000000004143
+:040178000000005C27
+:040179000000000F73
+:04017A000000000879
+:04017B000000003050
+:04017C00000000423D
 :04017D00000000631B
 :04017E000000006419
 :04017F000000006418
processing file asmFiles/beq.asm (1 of 24)
processing file asmFiles/beq_nt.asm (2 of 24)
processing file asmFiles/bne_nt.asm (3 of 24)
processing file asmFiles/bne_taken.asm (4 of 24)
processing file asmFiles/branch.asm (5 of 24)
processing file asmFiles/control_flow_hazard.asm (6 of 24)
processing file asmFiles/count_days.asm (7 of 24)
processing file asmFiles/empty.asm (8 of 24)
processing file asmFiles/fib.asm (9 of 24)
processing file asmFiles/forward.asm (10 of 24)
processing file asmFiles/halt.asm (11 of 24)
processing file asmFiles/jreg.asm (12 of 24)
processing file asmFiles/jump.asm (13 of 24)
processing file asmFiles/ld_hazard.asm (14 of 24)
processing file asmFiles/mergesort.asm (15 of 24)
--- memsim.hex	2022-02-23 10:39:52.115418222 -0500
+++ memcpu.hex	2022-02-23 10:39:52.036415265 -0500
@@ -81,131 +81,131 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
+:0400C1000000000338
+:0400C2000000003802
+:0400C3000000003DFC
+:0400C4000000000434
+:0400C5000000003601
+:0400C6000000000135
+:0400C7000000002015
+:0400C8000000003DF7
+:0400C90000000042F1
+:0400CA000000001B17
+:0400CB000000002110
+:0400CC000000000729
+:0400CD000000001D12
+:0400CE000000000B23
+:0400CF000000001617
+:0400D0000000004DDF
+:0400D10000000051DA
+:0400D20000000053D7
+:0400D30000000047E2
+:0400D4000000004BDD
 :0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
+:0400D6000000000C1A
+:0400D7000000001510
+:0400D800000000160E
+:0400D90000000029FA
 :0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400DB000000001908
+:0400DC0000000033ED
+:0400DD0000000051CE
 :0400DE000000005AC4
 :0400DF0000000063BA
 :0400E00000000064B8
-:0400E1000000000714
+:0400E10000000043D8
 :0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
+:0400E30000000032E7
+:0400E4000000001008
+:0400E50000000026F1
 :0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
+:0400E70000000026EF
+:0400E80000000024F0
+:0400E900000000080B
+:0400EA000000001AF8
+:0400EB0000000028E9
+:0400EC000000005BB5
+:0400ED000000005CB3
+:0400EE0000000017F7
+:0400EF000000000706
+:0400F0000000000A02
+:0400F10000000020EB
+:0400F20000000044C6
+:0400F30000000023E6
+:0400F4000000000701
+:0400F50000000011F6
+:0400F60000000059AD
+:0400F7000000002FD6
+:0400F80000000009FB
+:0400F90000000020E3
+:0400FA0000000041C1
+:0400FB000000005CA5
+:0400FC000000000FF1
+:0400FD0000000008F7
+:0400FE0000000030CE
+:0400FF0000000042BB
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
+:0401400000000003B8
+:040141000000003882
+:040142000000003D7C
+:0401430000000004B4
+:040144000000003681
+:0401450000000001B5
+:040146000000002095
+:040147000000003D77
+:040148000000004271
+:040149000000001B97
+:04014A000000002190
+:04014B0000000007A9
+:04014C000000001D92
+:04014D000000000BA3
+:04014E000000001697
+:04014F000000004369
+:0401500000000007A4
+:040151000000003278
+:040152000000001099
+:040153000000002682
+:04015400000000089F
+:040155000000002680
+:040156000000002481
+:04015700000000089C
+:040158000000001A89
+:04015900000000287A
+:04015A000000004D54
+:04015B00000000514F
+:04015C00000000534C
+:04015D000000004757
+:04015E000000004B52
+:04015F000000003E5E
+:040160000000000C8F
+:040161000000001585
+:040162000000001683
+:04016300000000296F
+:040164000000005047
+:04016500000000197D
 :040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
+:040167000000005143
+:040168000000005A39
+:040169000000005B37
+:04016A000000005C35
+:04016B000000001779
+:04016C000000000788
+:04016D000000000A84
+:04016E00000000206D
+:04016F000000004448
+:040170000000002368
+:040171000000000783
+:040172000000001178
+:04017300000000592F
+:040174000000002F58
+:04017500000000097D
+:040176000000002065
+:040177000000004143
+:040178000000005C27
+:040179000000000F73
+:04017A000000000879
+:04017B000000003050
+:04017C00000000423D
 :04017D00000000631B
 :04017E000000006419
 :04017F000000006418
processing file asmFiles/mult.asm (16 of 24)
processing file asmFiles/mult_procedure.asm (17 of 24)
processing file asmFiles/search.asm (18 of 24)
--- memsim.hex	2022-02-23 10:39:57.208608870 -0500
+++ memcpu.hex	2022-02-23 10:39:57.124605726 -0500
@@ -14,7 +14,6 @@
 :04000D001000FFF9E7
 :04000E00AFA500009A
 :04000F00FFFFFFFFF1
-:0400200000000124B7
 :0400210000005C6F10
 :040022000000006476
 :040023000000087D54
processing file asmFiles/test.branchjump1.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
processing file asmFiles/try.asm (24 of 24)
processing file asmFiles/search.asm (1 of 1)
--- memsim.hex	2022-02-23 10:40:38.828166773 -0500
+++ memcpu.hex	2022-02-23 10:40:38.754164003 -0500
@@ -14,7 +14,6 @@
 :04000D001000FFF9E7
 :04000E00AFA500009A
 :04000F00FFFFFFFFF1
-:0400200000000124B7
 :0400210000005C6F10
 :040022000000006476
 :040023000000087D54
processing file asmFiles/mergesort.asm (1 of 1)
--- memsim.hex	2022-02-23 10:49:55.843019248 -0500
+++ memcpu.hex	2022-02-23 10:49:55.737015279 -0500
@@ -81,131 +81,131 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
+:0400C1000000000338
+:0400C2000000003802
+:0400C3000000003DFC
+:0400C4000000000434
+:0400C5000000003601
+:0400C6000000000135
+:0400C7000000002015
+:0400C8000000003DF7
+:0400C90000000042F1
+:0400CA000000001B17
+:0400CB000000002110
+:0400CC000000000729
+:0400CD000000001D12
+:0400CE000000000B23
+:0400CF000000001617
+:0400D0000000004DDF
+:0400D10000000051DA
+:0400D20000000053D7
+:0400D30000000047E2
+:0400D4000000004BDD
 :0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
+:0400D6000000000C1A
+:0400D7000000001510
+:0400D800000000160E
+:0400D90000000029FA
 :0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400DB000000001908
+:0400DC0000000033ED
+:0400DD0000000051CE
 :0400DE000000005AC4
 :0400DF0000000063BA
 :0400E00000000064B8
-:0400E1000000000714
+:0400E10000000043D8
 :0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
+:0400E30000000032E7
+:0400E4000000001008
+:0400E50000000026F1
 :0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
+:0400E70000000026EF
+:0400E80000000024F0
+:0400E900000000080B
+:0400EA000000001AF8
+:0400EB0000000028E9
+:0400EC000000005BB5
+:0400ED000000005CB3
+:0400EE0000000017F7
+:0400EF000000000706
+:0400F0000000000A02
+:0400F10000000020EB
+:0400F20000000044C6
+:0400F30000000023E6
+:0400F4000000000701
+:0400F50000000011F6
+:0400F60000000059AD
+:0400F7000000002FD6
+:0400F80000000009FB
+:0400F90000000020E3
+:0400FA0000000041C1
+:0400FB000000005CA5
+:0400FC000000000FF1
+:0400FD0000000008F7
+:0400FE0000000030CE
+:0400FF0000000042BB
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
+:0401400000000003B8
+:040141000000003882
+:040142000000003D7C
+:0401430000000004B4
+:040144000000003681
+:0401450000000001B5
+:040146000000002095
+:040147000000003D77
+:040148000000004271
+:040149000000001B97
+:04014A000000002190
+:04014B0000000007A9
+:04014C000000001D92
+:04014D000000000BA3
+:04014E000000001697
+:04014F000000004369
+:0401500000000007A4
+:040151000000003278
+:040152000000001099
+:040153000000002682
+:04015400000000089F
+:040155000000002680
+:040156000000002481
+:04015700000000089C
+:040158000000001A89
+:04015900000000287A
+:04015A000000004D54
+:04015B00000000514F
+:04015C00000000534C
+:04015D000000004757
+:04015E000000004B52
+:04015F000000003E5E
+:040160000000000C8F
+:040161000000001585
+:040162000000001683
+:04016300000000296F
+:040164000000005047
+:04016500000000197D
 :040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
+:040167000000005143
+:040168000000005A39
+:040169000000005B37
+:04016A000000005C35
+:04016B000000001779
+:04016C000000000788
+:04016D000000000A84
+:04016E00000000206D
+:04016F000000004448
+:040170000000002368
+:040171000000000783
+:040172000000001178
+:04017300000000592F
+:040174000000002F58
+:04017500000000097D
+:040176000000002065
+:040177000000004143
+:040178000000005C27
+:040179000000000F73
+:04017A000000000879
+:04017B000000003050
+:04017C00000000423D
 :04017D00000000631B
 :04017E000000006419
 :04017F000000006418
processing file asmFiles/control_flow_hazard.asm (1 of 1)
--- memsim.hex	2022-02-23 10:56:39.945148409 -0500
+++ memcpu.hex	2022-02-23 10:56:39.828144028 -0500
@@ -9,7 +9,7 @@
 :040008003405012397
 :04000900AC650000E2
 :04000A00FFFFFFFFF6
-:04000F0000000123C9
+:04000F0000000BAD35
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
processing file asmFiles/control_flow_hazard.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/beq.asm (1 of 24)
processing file asmFiles/beq_nt.asm (2 of 24)
processing file asmFiles/bne_nt.asm (3 of 24)
processing file asmFiles/bne_taken.asm (4 of 24)
processing file asmFiles/branch.asm (5 of 24)
processing file asmFiles/control_flow_hazard.asm (6 of 24)
processing file asmFiles/count_days.asm (7 of 24)
processing file asmFiles/empty.asm (8 of 24)
processing file asmFiles/fib.asm (9 of 24)
processing file asmFiles/forward.asm (10 of 24)
processing file asmFiles/halt.asm (11 of 24)
processing file asmFiles/jreg.asm (12 of 24)
processing file asmFiles/jump.asm (13 of 24)
processing file asmFiles/ld_hazard.asm (14 of 24)
processing file asmFiles/mergesort.asm (15 of 24)
processing file asmFiles/mult.asm (16 of 24)
processing file asmFiles/mult_procedure.asm (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/test.branchjump1.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
processing file asmFiles/try.asm (24 of 24)
processing file asmFiles/beq.asm (1 of 24)
processing file asmFiles/beq_nt.asm (2 of 24)
processing file asmFiles/bne_nt.asm (3 of 24)
processing file asmFiles/bne_taken.asm (4 of 24)
processing file asmFiles/branch.asm (5 of 24)
processing file asmFiles/control_flow_hazard.asm (6 of 24)
processing file asmFiles/count_days.asm (7 of 24)
processing file asmFiles/empty.asm (8 of 24)
processing file asmFiles/fib.asm (9 of 24)
processing file asmFiles/forward.asm (10 of 24)
processing file asmFiles/halt.asm (11 of 24)
processing file asmFiles/jreg.asm (12 of 24)
processing file asmFiles/jump.asm (13 of 24)
processing file asmFiles/ld_hazard.asm (14 of 24)
processing file asmFiles/mergesort.asm (15 of 24)
processing file asmFiles/mult.asm (16 of 24)
processing file asmFiles/mult_procedure.asm (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/test.branchjump1.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
processing file asmFiles/try.asm (24 of 24)
processing file asmFiles/test.rtype.asm (1 of 1)
--- memsim.hex	2022-02-23 11:03:21.676189276 -0500
+++ memcpu.hex	2022-02-23 11:06:29.942238546 -0500
@@ -1,11 +1,44 @@
-:0400000034030004C1
-:0400010034010022A4
-:0400020034020022A2
-:04000300340500229E
-:04000400340800229A
-:040005008C6100000A
-:04000600340200229E
-:04000700340500229A
-:040008003408002296
-:04000900FFFFFFFFF7
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C00340E0004AA
+:04000D0001C15804D1
+:04000E00340E0005A7
+:04000F0001C16006C5
+:04001000002268273B
+:040011000022782A27
+:040012002830000290
+:040013000022882B14
+:040014002C32000288
+:04001500AECD00006C
+:04001600AEA3000095
+:04001700AEA400048F
+:04001800AEA5000889
+:04001900AEA6000C83
+:04001A00AEA700107D
+:04001B00AEA8001477
+:04001C00AEA9001871
+:04001D00AEAA001C6B
+:04001E00AEAB002065
+:04001F00AEAC00245F
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/beq.asm (1 of 24)
processing file asmFiles/beq_nt.asm (2 of 24)
processing file asmFiles/bne_nt.asm (3 of 24)
processing file asmFiles/bne_taken.asm (4 of 24)
processing file asmFiles/branch.asm (5 of 24)
processing file asmFiles/control_flow_hazard.asm (6 of 24)
processing file asmFiles/count_days.asm (7 of 24)
processing file asmFiles/empty.asm (8 of 24)
processing file asmFiles/fib.asm (9 of 24)
processing file asmFiles/forward.asm (10 of 24)
processing file asmFiles/halt.asm (11 of 24)
processing file asmFiles/jreg.asm (12 of 24)
processing file asmFiles/jump.asm (13 of 24)
processing file asmFiles/ld_hazard.asm (14 of 24)
processing file asmFiles/mergesort.asm (15 of 24)
processing file asmFiles/mult.asm (16 of 24)
processing file asmFiles/mult_procedure.asm (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/test.branchjump1.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
--- memsim.hex	2022-02-23 11:13:41.858410521 -0500
+++ memcpu.hex	2022-02-23 11:13:43.530473120 -0500
@@ -1,20 +1,44 @@
-:04000000340100F0D7
-:040001003402008045
-:040002003C07DEAD2C
-:0400030034E7BEEF31
-:040004008C23000049
-:040005008C24000443
-:040006008C2500083D
-:04000700AC43000006
-:04000800AC44000400
-:04000900AC450008FA
-:04000A00AC47000CF3
-:04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C00340E0004AA
+:04000D0001C15804D1
+:04000E00340E0005A7
+:04000F0001C16006C5
+:04001000002268273B
+:040011000022782A27
+:040012002830000290
+:040013000022882B14
+:040014002C32000288
+:04001500AECD00006C
+:04001600AEA3000095
+:04001700AEA400048F
+:04001800AEA5000889
+:04001900AEA6000C83
+:04001A00AEA700107D
+:04001B00AEA8001477
+:04001C00AEA9001871
+:04001D00AEAA001C6B
+:04001E00AEAB002065
+:04001F00AEAC00245F
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/try.asm (24 of 24)
processing file asmFiles/test.rtype.asm (1 of 1)
--- memsim.hex	2022-02-23 11:15:02.690436730 -0500
+++ memcpu.hex	2022-02-23 11:15:42.944943788 -0500
@@ -1,11 +1,44 @@
-:0400000034030004C1
-:0400010034010022A4
-:0400020034020022A2
-:04000300340500229E
-:04000400340800229A
-:040005008C6100000A
-:04000600340200229E
-:04000700340500229A
-:040008003408002296
-:04000900FFFFFFFFF7
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C00340E0004AA
+:04000D0001C15804D1
+:04000E00340E0005A7
+:04000F0001C16006C5
+:04001000002268273B
+:040011000022782A27
+:040012002830000290
+:040013000022882B14
+:040014002C32000288
+:04001500AECD00006C
+:04001600AEA3000095
+:04001700AEA400048F
+:04001800AEA5000889
+:04001900AEA6000C83
+:04001A00AEA700107D
+:04001B00AEA8001477
+:04001C00AEA9001871
+:04001D00AEAA001C6B
+:04001E00AEAB002065
+:04001F00AEAC00245F
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/beq.asm (1 of 24)
processing file asmFiles/beq_nt.asm (2 of 24)
processing file asmFiles/bne_nt.asm (3 of 24)
processing file asmFiles/bne_taken.asm (4 of 24)
processing file asmFiles/branch.asm (5 of 24)
processing file asmFiles/control_flow_hazard.asm (6 of 24)
processing file asmFiles/count_days.asm (7 of 24)
processing file asmFiles/empty.asm (8 of 24)
processing file asmFiles/fib.asm (9 of 24)
processing file asmFiles/forward.asm (10 of 24)
processing file asmFiles/halt.asm (11 of 24)
processing file asmFiles/jreg.asm (12 of 24)
processing file asmFiles/jump.asm (13 of 24)
processing file asmFiles/ld_hazard.asm (14 of 24)
processing file asmFiles/mergesort.asm (15 of 24)
processing file asmFiles/mult.asm (16 of 24)
processing file asmFiles/mult_procedure.asm (17 of 24)
processing file asmFiles/search.asm (18 of 24)
processing file asmFiles/test.branchjump1.asm (19 of 24)
processing file asmFiles/test.ldst1.asm (20 of 24)
processing file asmFiles/test.ldst2.asm (21 of 24)
processing file asmFiles/test.loadstore.asm (22 of 24)
processing file asmFiles/test.rtype.asm (23 of 24)
--- memsim.hex	2022-02-23 13:25:39.529177129 -0500
+++ memcpu.hex	2022-02-23 13:26:14.762498024 -0500
@@ -1,20 +1,44 @@
-:04000000340100F0D7
-:040001003402008045
-:040002003C07DEAD2C
-:0400030034E7BEEF31
-:040004008C23000049
-:040005008C24000443
-:040006008C2500083D
-:04000700AC43000006
-:04000800AC44000400
-:04000900AC450008FA
-:04000A00AC47000CF3
-:04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C00340E0004AA
+:04000D0001C15804D1
+:04000E00340E0005A7
+:04000F0001C16006C5
+:04001000002268273B
+:040011000022782A27
+:040012002830000290
+:040013000022882B14
+:040014002C32000288
+:04001500AECD00006C
+:04001600AEA3000095
+:04001700AEA400048F
+:04001800AEA5000889
+:04001900AEA6000C83
+:04001A00AEA700107D
+:04001B00AEA8001477
+:04001C00AEA9001871
+:04001D00AEAA001C6B
+:04001E00AEAB002065
+:04001F00AEAC00245F
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/try.asm (24 of 24)
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/try.asm (1 of 1)
processing file asmFiles/try.asm (1 of 1)
